test/CMakeLists.txt

```diff
@@ -218,7 +218,20 @@ add_test(
             -i ${ROCM_PATH}/share/rocdecode/video/AMD_driving_virtual_20-AV1.ivf
 )
 
-# 12 - videoDecodeRaw VP9
+# 12 - videoDecodeRaw AV1 OBU
+add_test(
+  NAME
+    video_decodeRaw-AV1-OBU
+  COMMAND
+    "${CMAKE_CTEST_COMMAND}"
+            --build-and-test "${ROCM_PATH}/share/rocdecode/samples/videoDecodeRaw"
+                              "${CMAKE_CURRENT_BINARY_DIR}/videoDecodeRaw"
+            --build-generator "${CMAKE_GENERATOR}"
+            --test-command "videodecoderaw"
+            -i ${ROCM_PATH}/share/rocdecode/video/AMD_driving_virtual_20-AV1.obu
```

Comment:
@jeffqjiangNew, This test is failing on math-CI because the file `AMD_driving_virtual_20-AV1.obu` is missing. Please upload the file to the `data/videos` folder.